examples/basic/basic.c

```diff
@@ -24,8 +24,7 @@ struct async_print_data {
 };
 
 struct async_print_output {
-	/* XXX dummy field to avoid empty struct */
-	uintptr_t foo;
+	int error_code;
```

Comment:
I was thinking, wouldn't the `return_code` be a better name? Since, if I understand correctly, this value doesn't always indicate an error.